bool ack
int32 markerId
float32 l_pixel
float32 centerDistance
